keyboards/technicpad/info.json

```diff
@@ -0,0 +1,34 @@
+{
+    "manufacturer": "nwhirschfeld",
+    "keyboard_name": "TechnicPad",
+    "maintainer": "nwhirschfeld",
+    "bootloader": "caterina",
+    "bootmagic": {
+        "matrix": [3, 0]
```

Comment:
This does not match the comment within the readme.

> * **Bootmagic reset**: Hold down the upper left key and plug in the keyboard

Reply:
resolved in https://github.com/nwhirschfeld/qmk_firmware/commit/7cba09e585de78caee4bd05af1534f4a54ef25a3